extractor/src/test/resources/expected-config-all-in-one.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
```

Comment:
Such files should substitute all files https://github.com/checkstyle/test-configs/tree/main/extractor/src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInName/Configs

Please copy such files to this folder.

Reply:
replaced all files in the above path